src/myrtlespeech/model/deep_speech_1.py

```diff
@@ -128,16 +134,18 @@ def forward(
                 represents the sequence length of the corresponding *input*
                 sequence.
 
-        Returns:
-            The first element of the Tuple return value is the result after
-            applying the model to ``x[0]``. It must have size ``[seq_len,
-            batch, out_features]``.
+            hx: An Optional RNNState of ``self.bi_lstm`` (see
+                :py:class:`RNN` for details.)
 
-            The second element of the Tuple return value is a
+        Returns:
+            A ``Tuple[Tuple[output, lengths], state]``. ``output`` is the
+            result after applying the model to ``x[0]``. It must have size
```

Comment:
`must have` is a weird statement for output that the user has no control over. Perhaps `will have`?

Reply:
done